packages/app/src/components/MenuButton/styles/MenuButton.module.css

```diff
@@ -0,0 +1,30 @@
+/* TODO: Fetch the colors from ui-core, once the same is done for the FileMenu (@dhis2/d2-ui-file-menu)  */
+
+.menuButton {
+    display: inline-flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+    font-size: 15px;
+    font-weight: 400;
+    text-transform: none;
+    padding: 6px 8px;
+    color: #000000; /* var(--colors-grey900) */
```

Comment:
Is this because FileMenu has a different color?

Reply:
Yes 👍